main.tf

```diff
@@ -31,15 +39,55 @@ data "aws_sns_topic" "main" {
 # CloudWatch Event
 #
 
-resource "aws_cloudwatch_event_rule" "main" {
+resource "aws_cloudwatch_event_rule" "main_gov" {
+  count         = data.aws_partition.current.partition == "aws-us-gov" ? 1 : 0
+  name          = "iam-root-login"
+  description   = "Successful login with root account"
+  event_pattern = <<PATTERN
+  {
+    "detail": {
+        "userIdentity": {
+            "arn": [
+                "arn:aws-us-gov:iam::${data.aws_caller_identity.current.account_id}:user/Administrator"
+            ],
+            "type": [
+                "IAMUser"
+            ]
+        },
+        "eventType": [
+            "AwsConsoleSignIn"
+        ]
+    },
+    "detail-type": [
+      "AWS Console Sign In via CloudTrail"
+    ]
+  }
+  PATTERN
+}
+
+resource "aws_cloudwatch_event_rule" "main_com" {
+  count         = data.aws_partition.current.partition == "aws" ? 1 : 0
```

Comment:
Only question I have is if it makes sense to distinguish the commercial and govcloud names to it's easier to distinguish when they fire into Slack.

Reply:
YES PLEASE!